src/pages/Home.jsx

```diff
@@ -46,8 +48,14 @@ class Home extends React.Component {
           <CategoryBar />
         </section>
         <section className="result-page">
-          <Input onChange={this.onChangeHandle} />
-          <button onClick={this.onClickHandle}>&#9740;</button>
+          <Input onChange={ this.onChangeHandle } />
+          <button
+            data-testid="query-button"
+            type="submit"
+            onClick={ this.onClickHandle }
+          >
+            &#9740;
+          </button>
           <p className="text-search" data-testid="home-initial-message">
             Digite algum termo de Pesquisa ou escolha uma categoria.
```

Comment:
o p em pesquisa está maiúsculo, por isso não passou no requisito 2.
```suggestion
            Digite algum termo de pesquisa ou escolha uma categoria.
```